yarn.lock

```diff
@@ -1786,8 +1786,8 @@
     react-transition-group "^4.4.5"
 
 "@mui/monorepo@https://github.com/mui/material-ui.git#master":
-  version "5.14.2"
-  resolved "https://github.com/mui/material-ui.git#5bac8b3c66ae144c35609129e8391c1653bf0aae"
+  version "5.14.3"
```

Comment:
There's still no link to the `sx` page in https://deploy-preview-9890--material-ui-x.netlify.app/x/api/data-grid/data-grid/#DataGrid-prop-sx
Am I missing something? 😅

Reply:
fixed

the `classes` seems to have no effect, I will have to check what was the initial text added

Reply:
It's because the name is `cssApi` not classes